src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -84,6 +84,10 @@
     <Compile Include="$(CommonPath)\System\Net\InternalException.cs">
       <Link>Common\System\Net\InternalException.cs</Link>
     </Compile>
+    <!-- HTTP -->
+    <Compile Include="..\..\Common\src\System\Net\Http\TlsCertificateExtensions.cs">
```

Comment:
Instead: `<Compile Include="$(CommonPath)\System\Net\Http\TlsCertificateExtensions.cs">`
(same as above and below in this file)